### ISSUE TITLE
Add softmax numerical stability for extreme logits

### ISSUE THREAD

**Issue by johannesdwicahyo** [johannesdwicahyo/onnx-ruby#8]
## Problem
`Classifier#softmax` subtracts max before exp, which is good, but doesn't guard against NaN/Inf results when logits contain NaN or Inf values.

## Fix
Add NaN/Inf check on input logits and output probabilities.

## Location
`lib/onnx_ruby/classifier.rb:97-101`

**Comment by johannesdwicahyo** [comment]
Implemented in v0.2.0
